sound/soc/sof/pcm.c

```diff
@@ -675,6 +675,9 @@ static int sof_pcm_dai_link_fixup(struct snd_soc_pcm_runtime *rtd,
 	case SOF_DAI_INTEL_HDA:
 		/* do nothing for HDA dai_link */
 		break;
+	case SOF_DAI_INTEL_ALH:
+		/* do nothing for ALH dai_link */
```

Comment:
@plbossart I think removing the break and allowing to fall through would look better but its just my personal choice. No need to change if you feel otw